wordpress-mobile/WordPress-Android
# Drag and Drop into the editor in split-view mode

Fixes #4378 

To test:

* Have an Android 7.0 emulator set up
* Put wpandroid in a split-view setup (e.g. when in the Recent Apps List, drag the title of the app and put it on the left edge of the emulator)
* Open Chrome in the right-side panel
* In wpandroid, go into the post editor
* In Chrome, type some address in the address bar, select it and drag-and-drop it into the editor
* The selected text (URL) should now be inserted into the editor, at the position it was dropped

Same procedure is supported for dropping images, though a gallery app that supports drag-and-drop needs to be installed.


Needs review: Hey @aforcier , want to have a look at this one? Thanks!

/cc @oguzkocer
